server/routes/api/user.js

```diff
@@ -1,7 +1,47 @@
 const express = require('express');
-
 const router = express.Router();
+const bcrypt = require('bcrypt');
+
+// User Model
+const User = require('../../models/user');
+
+// Test User route
+router.get('/test', (req, res) => res.json({ msg: 'User route works!' }));
+
+// POST User Registration
+router.post('/register', (req, res) => {
+  const email = req.body.email.trim();
+  const password = req.body.password.trim();
+
+  User.findOne({ email: email }).then(user => {
+    if (user) {
+      return res.status(400).json({ email: 'Email already exists.' });
+    } else if (!password) {
+      return res.status(400).json({ password: 'You must provide a password.' });
```

Comment:
I actually have it set up to test with the '.' so I've updated the requirements. Good job!